src/components/Calculator.js

```diff
@@ -1,36 +1,62 @@
 import React from 'react';
 import './index.css';
+import calculate from '../logic/calculate';
+// import operate from '../logic/operate';
```

Comment:
### OPTIONAL

- [ ] operate.js is already being used by calculate.js, so there's no need to import it. please remove commented code 👍🏿